app/models/user.rb

```diff
@@ -14,4 +14,8 @@ def check_if_first_user
     return true
   end
 
```

Comment:
Does the gem require you implement this method?  If not, I think it might not be used.